velox/dwio/parquet/writer/Writer.h

```diff
@@ -94,6 +94,7 @@ struct WriterOptions {
   // The default factory allows the writer to construct the default flush
   // policy with the configs in its ctor.
   std::function<std::unique_ptr<DefaultFlushPolicy>()> flushPolicyFactory;
+  std::shared_ptr<const Type> schema;
```

Comment:
TypePtr

Would you document this field? Why would we want to allow it to be null?